concepticondata/conceptlists.tsv

```diff
@@ -293,13 +293,13 @@ Schroeder-2012-824	Schroeder, Astrid and Gemballa, Teresa and Ruppin, Steffie an
 Hsiu-2015-315	Hsiu, Andrew C.	2015		315	areal, questionnaire	English	Hmong-Mien languages	https://zenodo.org/record/1127804/	Hsiu2015		Hmong-Mien languages in Vietnam. The dataset contains 12 Varieties and 315 concepts.		
-Maciejewski-2016-100	Maciejewski, Greg and Klepousniotou, Ekaterini	2016		100	ratings	English	English	 http://osf.io/7k3eh/	Maciejewski2016		The list comprises relative meaning frequencies for 100 homonyms (British eDom Norms) and other lexical norms. For feasibility, each line of the table was duplicated to map the different meanings to one of the concepts found in Concepticon. The meaning definition was taken from the Wordsmyth dictionary [(Parks et al. 1998)](:bib:Parks1998). The dominance scores were included from [Armstrong et al. (2012)](:bib:Armstrong2012). Word frequencies and number of orthographic neighbours  were based on the British National Corpus [(BNC 2007)](:bib:BNC2007). The numbers for word senses were taken from [Parks et al. (1998)](:bib:Parks1998) and WordNet [(Fellbaum 1998)](:bib:Fellbaum1998). Semantic diversity values were included from [(Hoffman et al. 2013)](:bib:Hoffman2013). Ratings on familiarity, concreteness, and imageability were taken from [Stadthagen-Gonzalez and Davis (2006)](:bib:Stadthagen2006) and [Coltheart (1981)](:bib:Coltheart1981).		
+Maciejewski-2016-100	Maciejewski, Greg and Klepousniotou, Ekaterini	2016		100	ratings	English	English	" http://osf.io/7k3eh/"	Maciejewski2016		The list comprises relative meaning frequencies for 100 homonyms (British eDom Norms) and other lexical norms. For feasibility, each line of the table was duplicated to map the different meanings to one of the concepts found in Concepticon. The meaning definition was taken from the Wordsmyth dictionary [(Parks et al. 1998)](:bib:Parks1998). The dominance scores were included from [Armstrong et al. (2012)](:bib:Armstrong2012). Word frequencies and number of orthographic neighbours  were based on the British National Corpus [(BNC 2007)](:bib:BNC2007). The numbers for word senses were taken from [Parks et al. (1998)](:bib:Parks1998) and WordNet [(Fellbaum 1998)](:bib:Fellbaum1998). Semantic diversity values were included from [(Hoffman et al. 2013)](:bib:Hoffman2013). Ratings on familiarity, concreteness, and imageability were taken from [Stadthagen-Gonzalez and Davis (2006)](:bib:Stadthagen2006) and [Coltheart (1981)](:bib:Coltheart1981).		
```

Comment:
There are some cases of `"` which should not be there in the file, you can see this from the DIFF here. Originally : 
```
http://osf.io/7k3eh/
```

now

```
" http://osf.io/7k3eh/"
```
This should be reverted.